core/src/Cuda/Kokkos_Cuda_Parallel_Range.hpp

```diff
@@ -987,10 +1021,11 @@ class ParallelScanWithTotal<FunctorType, Kokkos::RangePolicy<Traits...>,
       // How many block are really needed for this much work:
       const int grid_x = (nwork + work_per_block - 1) / work_per_block;
 
-      m_scratch_space = cuda_internal_scratch_space(
-          m_policy.space(), Analysis::value_size(m_functor) * grid_x);
-      m_scratch_flags =
-          cuda_internal_scratch_flags(m_policy.space(), sizeof(size_type) * 1);
+      m_scratch_space =
+          reinterpret_cast<word_size_type*>(cuda_internal_scratch_space(
+              m_policy.space(), Analysis::value_size(m_functor) * grid_x));
+      m_scratch_flags = cuda_internal_scratch_flags(
+          m_policy.space(), sizeof(Cuda::size_type) * 1);
```

Comment:
```suggestion
      m_scratch_flags =
          cuda_internal_scratch_flags(m_policy.space(), sizeof(size_type) * 1);
```

This change seems unnecessary. Doesn't change anything functionally, but I suggest reverting back to using `sizeof(size_type)` here to keep it consistent with other `[backend]_internal_scratch_flags` calls.

Reply:
Agreed.  I missed that one.  Will revert.